### PR TITLE
[WIP DNR] Stack View Layout Rewrite for #58?

### PR DESCRIPTION
**Pull request by kyleve**
Don't review this please

### CODE REVIEW

File: BlueprintUI/Sources/Layout/Stack.swift [hunk 1]
```diff
@@ -119,14 +119,140 @@ public struct StackLayout: Layout {
     }
 
     public func measure(in constraint: SizeConstraint, items: [(traits: Traits, content: Measurable)]) -> CGSize {
```
Comment (kyleve) [inline BlueprintUI/Sources/Layout/Stack.swift]: This is wrong in the limit, but fine for me RN

**Comment by bizzemfrog** [comment]
Is there anything I can do to help this along?

**Comment by kyleve** [comment]
I hit some roadblocks getting this working, so I ended up putting it on the backburner – it was also very much in prototype state, and probably isnt something I actually want to merge... @watt or @kylebshr have either of you started digging into this as well?

**Comment by watt** [comment]
Yeah, a bit. I'll see what I can put together. 👍

**Pull request closed without merging**